contracts/test/UntronFees.t.sol

```diff
@@ -103,8 +103,7 @@ function test_SetZeroAndNegativeFees(int256 relayerFee, int256 fulfillerFee) pub
 vm.startPrank(admin);
     if (relayerFee >= 0 && fulfillerFee >= 0) {
         untronFees.setFeesVariables(uint256(relayerFee), uint256(fulfillerFee));
-    } else {
-        vm.expectRevert();
+    } else { 
```

Comment:
So a couple of things here
- First, since `relayerFee` and `fulfillerFee` are actually `uint256` values, then mapping a negative int to them doesnt make sense (the `uint256` will be positive). Thus, there is no need to check for "negative fees", they dont existe since they are always interpreted as positive

This leaves us with only one case to test which should be: `test_setFeesVariables_RevertIf_FeeIsZero`

Reply:
The relayerFees and fulfillerFee are int256 from the (your) former fuzz test. I will change that to uint256.  and fix the new test as stated above.

Reply:
You can see that the else clause does the same as the if clause, thus the if/else clause is not needed
Apart from that, in the test you are checking that the fee is actually zero. So you can just do instead

```
function test_setFeesVariables_RevertIf_FeeIsZero() public {
   uint256 relayerFee = 0;
   uint256 fulfillerFee = 0;
   
   ...
```